pontoon/base/models.py

```diff
@@ -25,12 +25,77 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
```

Comment:
Oh, I like the idea of having a separate manager for this! I've never actually tried using specialized managers like this before so I'm interested in checking it out.